lib/Wikia/src/Service/User/Auth/HeliosCookieHelper.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Wikia\Service\User\Auth;
+
+use Wikia\HTTP\Response;
+use Wikia\Service\Helios\HeliosClient;
+
+class HeliosCookieHelper implements CookieHelper {
+
+	const ACCESS_TOKEN_COOKIE_NAME = 'access_token';
+	// This is set to 6 months,(365/2)*24*60*60 = 15768000
+	const ACCESS_TOKEN_COOKIE_TTL = 15768000;
+	const COOKIE_PREFIX = '';
```

Comment:
Having the same constants and methods in many places doesn't help in future development. Current state is really confusing - there is no ```@Deprecated``` annotation in original class, code is still present in original class.